### DISCUSSION TITLE
ajouter un article spécifique sur les cotisations en tant qu'artistes-autrices en intégrant direct le module de mon-entreprise

### DISCUSSION THREAD

**Discussion by JenniferStephan** [betagouv/ellesfontlaphoto#241] [Ideas]
https://mon-entreprise.urssaf.fr/simulateurs/artiste-auteur

+ le guide ursaff ? https://www.secu-artistes-auteurs.fr/sites/default/files/pdf/Guide%20pratique%20de%20d%c3%a9but%20d'activit%c3%a9.pdf

**Comment by wittsarah** [comment]
Merci @JenniferStephan super idée ! On partage ça avec Netscouade pour l'ajouter en ressources utiles et peut être en faire un article dédié :)
